client/app/bundles/course/admin/pages/LessonPlanSettings/MilestoneGroupSettings.jsx

```diff
@@ -54,13 +55,11 @@ class MilestoneGroupSettings extends Component {
 
   render() {
     return (
-      <>
-        <h2>
-          <FormattedMessage {...translations.header} />
-        </h2>
-        <p>
-          <FormattedMessage {...translations.explanation} />
-        </p>
+      <Subsection
+        sticksToNavbar
```

Comment:
I think you meant to use `Section` here? There's no `sticksToNavbar` in `Subsection.`

Reply:
Ah no. I am supposed to add sticksToNavBar to `Section` in `client/app/bundles/course/admin/pages/LessonPlanSettings/index.jsx`